### PR TITLE
Add missing line and improve wording

### PR DESCRIPTION
**Pull request by bszwarc**
**Description**

Changes proposed in this pull request:

- add missing line about **externalLink**
- improve wording and fix typos

**Related issue(s)**
See also #198

### CODE REVIEW

File: docs/navigation-configuration.md [hunk 1]
```diff
@@ -218,6 +218,9 @@ The node parameters are as follows:
 - **pathSegment** specifies the partial URL of the current segment. **pathSegment** must not contain slashes.
   - A static settings example reflects `luigidomain.test/settings`.
   - A dynamic settings example, prefixed with a colon, loads on any other value. 
+- **externalLink** is an object which indicates that the node links to an external URL. If this parameter is defined, **pathSegment** is ignored. **externalLink**  has the following properties:
```
Comment (tomekpapiernik) [inline docs/navigation-configuration.md]:
```suggestion
- **externalLink** is an object which indicates that the node links to an external URL. If this parameter is defined, **pathSegment** is ignored. It has the following properties:
```

Comment (tomekpapiernik) [inline docs/navigation-configuration.md]: I'd avoid repeating the name formatted in bold just for clarity.

File: docs/navigation-configuration.md [hunk 2]
```diff
@@ -226,8 +229,8 @@ The node parameters are as follows:
...
-- **loadingIndicator.enabled** shows a loading indicator when switching between micro front-ends. If you have a super fast micro front-end, you can disable this feature to prevent the flickering of the loading indicator. The **loadingIndicator.loadingIndicator** is enabled by default.
-- **loadingIndicator.hideAutomatically** disables the automatic hiding of the loading indicator once the micro front-end is loaded. Only considered if the loading indicator is enabled. Does not apply if the loading indicator is activated manually with the `LuigiClient.uxManager().showLoadingIndicator()` function. If the loading indicator is enabled and automatic hiding is disabled, use `LuigiClient.uxManager().hideLoadingIndicator()` to hide it manually in your micro front-end during the startup. The **loadingIndicator.hideAutomatically** is enabled by default.
+- **loadingIndicator.enabled** shows a loading indicator when switching between micro front-ends. If you have a fast micro front-end, you can disable this feature to prevent flickering of the loading indicator. The **loadingIndicator.loadingIndicator** is enabled by default.
```
Comment (tomekpapiernik) [inline docs/navigation-configuration.md]: The **loadingIndicator.loadingIndicator** is enabled by default. - should there really be 2x loadingindictaor? Or is this just a bug ?

**Comment by bszwarc** [comment]
closes #198

**Pull request merged**